### PR TITLE
Api reference fix brands merge. Use account id as ref v2

### PR DESCRIPTION
**Pull request by maksymmateiuk**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

**Review by ajrice6713 — DISMISSED** [review]

File: site/specs-temp/numbers.json [hunk 1]
```diff
@@ -19719,6 +17920,9 @@
         "operationId": "RetrieveSiteForThatTn",
         "summary": "Retrieve site for that tn",
         "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:17924]: This endpoint doesn't have account in the path, why are you adding it here?

Comment (maksymmateiuk) [reply site/specs-temp/numbers.json:17924]: Thank you! rechecked all places where we don't have account id in the path and removed there.

File: site/specs-temp/numbers.json [hunk 2]
```diff
@@ -20827,56 +19022,6 @@
         "type": "string",
         "maxLength": 255
       },
-      "LnpOrderResponse": {
```
Comment (mrmeyers99) [inline site/specs-temp/numbers.json:20830]: this isn't used anywhere?

**Review by ajrice6713 — APPROVED** [review]

File: site/specs-temp/numbers.json (hunk 2 above)
Comment (maksymmateiuk) [reply site/specs-temp/numbers.json:20830]: It's a duplicate.

**Review by mrmeyers99 — APPROVED** [review]

**Pull request merged**